docs/website/developers/committers/release-process/publish.md

```diff
@@ -69,12 +74,15 @@ link in the top right (the credentials are the same as your Git and Jenkins cred
 Repositories" page, and tick the repository with the name starting `orgapachebrooklyn`. Click the **Release** button.
 Provide a description which includes the version, e.g. `Apache Brooklyn 0.7.0-incubating`.
 
+Note the staging repository may have a name like "080 RC2". You can double-check under the "content" that brooklyn-dist
```

Comment:
The name will be whatever the RM gave in https://brooklyn.incubator.apache.org/developers/committers/release-process/publish-temp.html#close-the-staging-repository-on-apaches-nexus-server. That page suggested using a name like `Apache Brooklyn 0.7.0-incubating-rc1`; Alex seems to have picked something much more terse.

I'd remove this change; I suggested you do it since the RM changed partway through the release.